lib/min_heap.rb

```diff
@@ -39,33 +45,48 @@ def to_s
     end
 
     output += @store.last.value + "]"
-      
-    return output
+    output
   end
 
   # This method returns true if the heap is empty
-  # Time complexity: ?
-  # Space complexity: ?
+  # Time complexity: O(1)
+  # Space complexity: O(1)
   def empty?
-    raise NotImplementedError, "Method not implemented yet..."
+    @store.length.zero? ? true : false
```

Comment:
Simplification, you don't need the ternary

```suggestion
    return @store.length.zero?
```